devtools/server/actors/replay/react-devtools/contentScript.js

```diff
@@ -25,8 +25,17 @@ function initialize(dbgWindow, RecordReplayControl) {
       );
     };
 
+  window.wrappedJSObject.__RECORD_REPLAY_REACT_DEVTOOLS_HOOK__ = kind => {
+    RecordReplayControl.onAnnotation("react-devtools-hook", kind);
+  };
+
+  // The hook script is given a special URL so it won't be ignored and clients can
+  // inspect state in its frames. We test for this script in a few other places
+  // in gecko, so be careful when changing this.
```

Comment:
Do we really want clients to be able to inspect its state? I think from a user perspective it would be surprising to see a random script in every single recording.

If we do want to keep this, it seems like it would be better to give it a clear scheme like `replay:///react-devtools-hook` so we can group together any scripts we do inject in a clearly-defined group of replay content scripts. That would also the frontend to easily hide them if it wanted to.

Reply:
Actually, I think we might have uses for the replay:// protocol already (cc @ryanjduffy ), I'll use `replay-content://` to avoid potential confusion.